torch/csrc/jit/frontend/script_type_parser.cpp

```diff
@@ -243,6 +243,18 @@ TypePtr ScriptTypeParser::parseTypeFromExprImpl(const Expr& expr) const {
       return custom_class_type;
     }
 
+    if (type_name.find("torch.cuda.Stream") == 0) {
+        std::string class_type = "torch.classes.cuda.Stream";
+        auto custom_class_type = getCustomClass("__torch__." + class_type);
+        return custom_class_type;
+    }
+
+    if (type_name.find("torch.cuda.Event") == 0) {
+        std::string class_type = "torch.classes.cuda.Event";
+        auto custom_class_type = getCustomClass("__torch__." + class_type);
+        return custom_class_type;
+    }
+
```

Comment:
What's the error you are getting due to this?

One thing to check is whether this code above is resolving the type for these classes before they have a chance to execute:

```
      if (auto typePtr = resolver_->resolveType(type_name, expr.range())) {
        return typePtr;
      }
```

Reply:
Yeah, it is not able to resolve the type inside `stream`. It resolves to `torch.cuda.streams.Stream`, the eager mode `Stream`.

Reply:
I would check if the code I quoted above is resolving the type before the code you added is able to run.